library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -253,6 +250,10 @@ class SwipeActionView : FrameLayout {
             field = newResistance
         }
 
+    var resetDelay = 0L
+
+    var enableHapticFeedback = true
```

Comment:
Is this needed? I'm unfamiliar with haptic feedback on Android but there seems to be setHapticFeedbackEnabled built-in function that could be used instead. (Perhaps we just need to pass proper options for it to be respected when performing haptic feedback)

Reply:
I think you're right, I may have gotten carried away and started sticking all my changes in here instead of in my own code. I'll double check, but I can probably just remove it and put it in my code using the library.